src/docker-osx-dev

```diff
@@ -226,9 +225,9 @@ function do_rsync {
 #
 function sync {
   local readonly paths_to_sync=("$@")
-
-  for path in "${paths_to_sync[@]}"; do
-    do_rsync "$path"
+  for pathRsync in ${paths_to_sync[@]}
```

Comment:
Same for me, I guess it  may depends on the content of the array.

If its really a bug in all the project, I may search for the similar things and debug it + correct it.